core/templates/pages/exploration-editor-page/translation-tab/audio-translation-bar/audio-translation-bar.directive.ts

```diff
@@ -554,9 +554,8 @@ angular.module('oppia').directive('audioTranslationBar', [
               $scope.audioBlob = null;
             });
 
-            $scope.$on('activeContentIdChanged', function() {
-              $scope.initAudioBar();
-            });
+            TranslationTabActiveContentIdService.onActiveContentIdChange
```

Comment:
Create a new Subscription and unsubscribe as you did in the spec file. You can see how that is done here: https://github.com/oppia/oppia/pull/10070/files#diff-5b70c64139ce6f08f170ccaccf2505d4R40-R385

Reply:
Yes, changed it.